File: Domains/ApplicationDomain/BOA/Entities/FeedHistory.cs
using ApplicationDomain.Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.BOA.Entities
{
    public class FeedHistory : EntityBase<int>
    {
        public string Code { get; set; }
        public int CowId { get; set; }
        public Cow Cow { get; set; }

    }
}

File: Applications/WebAdminApplication/ClientApp/src/app/modules/farmer/models/feedHistory.ts
export class FeedHistory {
  id: number;
  code: string;
  cowId: number;
  createdDate: Date;

  constructor() {
   this.id =0;
   this.code="";
   this.cowId=0;
   this.createdDate = new Date();
  }
}
